Prepare `kedro-airflow` 0.5.0 release (#17)
kedro-org/kedro-plugins: kedro-airflow/RELEASE.md

# Release 0.5.0

> * Add compatibility with Kedro 0.18.0
> * Add compatibility with python 3.9 + 3.10
> * Drop support for python 3.6

Reviewer: ```suggestion
* Add compatibility with `kedro` 0.18.0
* Add compatibility with Python 3.9 and 3.10
* Remove compatibility with Python 3.6
```

Reviewer: Just to match the other releases.

> 
> ## Bugfix
> 
> ## Thanks for supporting contributions
> 

Reviewer: ```suggestion
```
